tests/api.c

```diff
@@ -65849,6 +65849,78 @@ static int test_dtls13_bad_epoch_ch(void)
 #endif
 
 
+#if defined(HAVE_NULL_CIPHER) && defined(HAVE_IO_TESTS_DEPENDENCIES) && \
+    defined(WOLFSSL_DTLS13)
+static byte* test_find_string(const char *string,
+    byte *buf, int buf_size)
+{
+    int string_size, i;
+
+    string_size = XSTRLEN(string);
+    for (i = 0; i < buf_size; i++) {
```

Comment:
If you do `i < buf_size - string_size - 1` you can skip the size check inside the loop.